Rakefile

```diff
@@ -15,4 +20,118 @@ else
   Spec::Rake::SpecTask.new(:spec)
 end
 
+desc "Update currency locales from rails-i18n repository"
```

Comment:
For completeness sake, I Included the rake task that Co-Pilot helped me with to auto update these locales from the upstream rails i18n gem. If we are ok with this, (somewhat hacky) rake task we can use it again in the future to add some automation to the updates to this gem.